src/main/java/seedu/tinner/logic/parser/CliSyntax.java

```diff
@@ -13,7 +13,7 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_STATUS = new Prefix("s/");
-    public static final Prefix PREFIX_DEADLINE = new Prefix("b/");
+    public static final Prefix PREFIX_REMINDER_DATE = new Prefix("b/");
```

Comment:
Perhaps we can change the prefix to `r/`? And somehow unify both company name and role name prefixes as `n/`.

Reply:
>Perhaps we can change the prefix to r/

I agree with Peter on this, makes more intuitive sense for the user if reminder date had the r/ prefix.

>unify both company name and role name prefixes as n/

I believe that these prefixes are for the find command where you need to specify both company and role name in the same command, hence the distinction between the two.

Reply:
I agree. Will make relevant changes.

Reply:
> I believe that these prefixes are for the find command where you need to specify both company and role name in the same command, hence the distinction between the two.


Ahhh yep forgot about the find command, using `n/` for both could definitely pose an issue then. Perhaps could use some other prefixes then.

Reply:
Yes I have retained the use of prefixes for the find command using c/ and r/. Only the addRole and editRole command has been edited, where the reminderDate field is indicated with a prefix r/.